lib/vagrant-service-manager.rb

```diff
@@ -8,7 +8,7 @@
 require 'vagrant-service-manager/command'
 require 'vagrant-service-manager/os'
 
-module Vagrant
+module VagrantPlugins
```

Comment:
Make sense, but do you do this for cosmetic reasons or is this required, for example for using the Vagrant logging capabilties?